src/components/bottom-sheet/dashboard-menu-bottom-sheet/styles.ts

```diff
@@ -16,13 +16,15 @@ export default (theme: ITheme) =>
         rowContainer: {
             flexDirection: 'row',
             paddingVertical: BASE_DIMENSION,
-            marginBottom: BASE_DIMENSION * 2
+            marginBottom: BASE_DIMENSION * 2,
+            alignItems: 'center'
         },
         iconContainer: {
             backgroundColor: theme.colors.appBackground,
             padding: BASE_DIMENSION,
             marginRight: BASE_DIMENSION * 2,
-            borderRadius: BORDER_RADIUS
+            borderRadius: BORDER_RADIUS,
+            maxHeight: BASE_DIMENSION * 6
```

Comment:
@NovacAndrei is there a problem with maxHeight ?

Reply:
Not exactly with maxHeight, but icons do not always have the same sizes (width and height) and because of that the parent views does not always look the same.

Look this for example:
![image](https://user-images.githubusercontent.com/32679726/94424654-3ec74880-0193-11eb-894f-be6d41dd49da.png)

If we enforce a width and height, remove paddings and aligns everything center, we shouldn't have any problems. On this icons we should not depend on paddings.